commands/copilot_cli.py

```diff
@@ -132,6 +132,7 @@ def make_cloudformation(storage_config_file, output, overwrite):
             "environments": environments,
             "prefix": camel_case(storage_name),
             "storage_type": storage_type,
+            "version": storage_config.get("version", None),
```

Comment:
This isn't needed because the line below will copy the field across if it's present

```
     **storage_config,
```

Reply:
Deleted